adafruit_io/adafruit_io.py

```diff
@@ -855,6 +855,23 @@ def receive_random_data(self, generator_id: int):
         path = self._compose_path("integrations/words/{0}".format(generator_id))
         return self._get(path)
 
+    def receive_user_info(self):
+        """
+        Get detailed account information for the current user.
+
+        See https://io.adafruit.com/api/docs/#get-user-info
+        """
+        return self._get("https://io.adafruit.com/api/v2/user")
+
+    def receive_user_rate_info(self):
```

Comment:
If we don't include this as an example, please add example output to the docstring so people know roughly what will happen when they call this method